docker/docker-compose.yaml

```diff
@@ -524,6 +524,9 @@ services:
     ports:
       - "80:80"
       #- "443:443"
+volumes:
+  postgres_data:
```

Comment:
Moving the volume directory will cause a Breaking change, please set the default directory to ./volumes for the specified directory.